Lib/uuid.py

```diff
@@ -182,6 +196,7 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             int &= ~(0xf000 << 64)
             int |= version << 76
         self.__dict__['int'] = int
+        self.__dict__['is_safe'] = is_safe
```

Comment:
I'm not sure it's worth type checking `is_safe`.  The other ones are checked to ensure sane, compliant UUIDs, but `is_safe` is just a flag with some extra information (which may or may not be helpful to an application).  If someone wants to manually instantiate UUIDs with bogus `is_safe` flags, oh well.